src/js/components/ExpandingWithCheckboxTable.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import CheckboxTable from './CheckboxTable';
+import ExpandingTable from './ExpandingTable';
+
+class ExpandingWithCheckboxTable extends React.Component {
+  getColGroup() {
+    return (
+      <colgroup>
+        <col style={{width: '40px'}} />
+        <col />
+        <col style={{width: '120px'}} className="hidden-mini" />
+        <col style={{width: '160px'}} className="hidden-mini" />
+        <col style={{width: '160px'}} className="hidden-mini" />
```

Comment:
honestly, after you take this method out, will this component even be necessary? it'll literally only have `render`